app/assets/javascripts/events.js

```diff
@@ -1,9 +1,81 @@
 // Place all the behaviors and hooks related to the matching controller here.
 // All this logic will automatically be available in application.js.
+idsOfInvitedUsers = new Array();
+namesOfInvitedUsers = new Array();
+
 document.addEventListener("turbolinks:load", function() {
+    idsOfInvitedUsers = new Array();
+    namesOfInvitedUsers = new Array();
 
     $('#calendar').fullCalendar(
         {events: '/events.json'}
     );
     $('#calendar_published_on').datepicker();
 });
+
+$(document).on('keyup', '.search-user-input_field', function (event) {
+    search = $(this).val();
+    search_user('../../user_search', on_search_user_results, search, 'layout_for_invites');
```

Comment:
suche Funktioniert nicht das Template _user_search_layout_for_invites.html.erb  existiert nicht, kann es nicht im commit finden. is das so gewollt, soll es trotzdem im Master?

```
ActionView::MissingTemplate (Missing template users/users/_user_search_layout_for_invites.html.erb with {:locale=>[:de], :formats=>[:html, :html, :text, :js, :css, :ics, :csv, :vcf, :png, :jpeg, :gif, :bmp, :tiff, :mpeg, :xml, :rss, :atom, :yaml, :multipart_form, :url_encoded_form, :json, :pdf, :zip], :variants=>[], :handlers=>[:erb, :builder, :raw, :ruby, :coffee, :jbuilder]}. Searched in:
  * "/Users/insanemac/Entwickl/PimRoR/app/views"
  * "/Users/insanemac/.rvm/gems/ruby-2.3.1/gems/devise-4.2.0/app/views"
):
  app/controllers/users/users_controller.rb:73:in `search'
```